include/tss2/tss2_esys.h

```diff
@@ -57,6 +57,14 @@ typedef uint32_t ESYS_TR;
 #define ESYS_TR_RH_ENDORSEMENT 0x10BU
 #define ESYS_TR_RH_PLATFORM    0x10CU
 #define ESYS_TR_RH_PLATFORM_NV 0x10DU
+#define ESYS_TR_RH_FW_OWNER       0x10EU
+#define ESYS_TR_RH_FW_ENDORSEMENT 0x10FU
+#define ESYS_TR_RH_FW_PLATFORM    0x110U
+#define ESYS_TR_RH_FW_NULL        0x111U
+#define ESYS_TR_RH_SVN_OWNER_BASE       0x1010000U
+#define ESYS_TR_RH_SVN_ENDORSEMENT_BASE 0x1020000U
+#define ESYS_TR_RH_SVN_PLATFORM_BASE    0x1030000U
+#define ESYS_TR_RH_SVN_NULL_BASE        0x1040000U
```

Comment:
AFAICT this fits nicely in the ESYS_TR "namespace" and won't conflict with the random() offset for non-permanent handles:
  - https://github.com/tpm2-software/tpm2-tss/blob/166a5538edb4eafaf64c7a45a63ab1a20ac30361/src/tss2-esys/esys_context.c#L79

This is more of a note for other reviewers to consider this than a comment on the patch.